error-handler.js

```diff
@@ -258,9 +258,17 @@ createHandler = function createHandler(options) {
         }
       },
 
+      shouldContinue = function
```

Comment:
OK, I'm willing to go along with removing this functionality. I guess I was thinking there could be some behavior where the 503 Too Busy would actually benefit from a process recycle. Removing this from the errorHandler puts that decision in the hands of the user handler.

To highlight that, I want to be sure you see the [usage](https://github.com/localnerve/express-error-handler/blob/b90babea3ce84c8eb97fd56aa8404b67d62dc208/README.md#example-1) of status().